client/cmd_auth_test.go

```diff
@@ -25,8 +25,8 @@ func TestClient_Select(t *testing.T) {
 	}()
 
 	tag, cmd := s.ScanCmd()
-	if cmd != "SELECT INBOX" {
-		t.Fatalf("client sent command %v, want SELECT INBOX", cmd)
+	if cmd != "SELECT \"INBOX\"" {
```

Comment:
```
select          = "SELECT" SP mailbox
mailbox         = "INBOX" / astring
```

Reply:
Hmm, so actually `INBOX` is a little annoying: I've seen e-mail servers and clients that don't like it when it's quoted. I guess we'll need to special-case this one (maybe with a `FormatMailbox` function in go-imap?).

Reply:
Done, see cc4b00e.